lib/kvs/kvs_cursor.c

```diff
@@ -1146,11 +1146,18 @@ kvs_cursor_val_copy(
     const void            **val_out,
     size_t                 *vlen_out)
 {
-    struct kvs_cursor_impl *cur = cursor_h2r(cursor);
+    struct kvs_cursor_impl *cur;
+    struct kvs_vtuple *     vt;
+    uint                    clen;
+    merr_t                  err = 0;
```

Comment:
I thought we agreed to stop lining up decls?

Reply:
I have no idea what the current conventions are.

Reply:
I haven't done it in ages, and I think Nabeel doesn't do it.  Maintenance headache and all..  Pretty sure you didn't like it, right?

Reply:
I do whatever I am told! I didn't really like it though.